src/ExpressionEvaluator/CSharp/Test/ResultProvider/DynamicViewTests.cs

```diff
@@ -0,0 +1,145 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Dynamic;
+using System.Linq;
+using Microsoft.CodeAnalysis.ExpressionEvaluator;
+using Microsoft.VisualStudio.Debugger.Clr;
+using Microsoft.VisualStudio.Debugger.Evaluation;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class DynamicViewTests : CSharpResultProviderTestBase
```

Comment:
***missing Microsoft.CSharp.dll:*** This is an implementation detail of InstantiateDynamicViewProxy.  We should have no knowledge of it, and we expect the method to return null if the assembly is not available.  I've refactored the tests so that they don't explicitly load Microsoft.CSharp (test mock now does it).

***member access returns Error value:*** Added ```DynamicViewTests.DynamicTypeError```

***proxy evaluation returns Error value:*** Added ```DynamicViewTests.DynamicMetaObjectProviderDebugViewItemsError```